### PR TITLE
Moves from gofmt to gofumpt

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Amazingly, this caught nothing.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -31,12 +32,8 @@ lint: $(golangci_lint_path)
 
 .PHONY: format
 format:
-	@find . -type f -name '*.go' | xargs gofmt -s -w
-	@for f in `find . -name '*.go'`; do \
-	    awk '/^import \($$/,/^\)$$/{if($$0=="")next}{print}' $$f > /tmp/fmt; \
```
Comment (anuraaga) [inline Makefile:36]: Does gofumpt handle the import grouping that I think this was for?

Comment (anuraaga) [reply Makefile:36]: Though if not, while on the note of swapping formatting tools, can consider gosimports

https://github.com/rinchsan/gosimports

Comment (codefromthecrypt) [reply Makefile:36]: I think gofumpt mainly groups stdlib imports, so I think you are right it wouldn't be able to correct dependencies vs our imports.

**Review by anuraaga — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
thanks for the tip!

**Pull request merged**